alembic.ini

```diff
@@ -2,7 +2,7 @@
 
 [alembic]
 # path to migration scripts
-script_location = migrations
+script_location = superset/migrations
```

Comment:
Why did the path change? Was this a tweak for dev reasons?

Reply:
alembic wasn't able to find the folder `migrations` without this, though I could just change it back before making the real PR if I was just using alembic incorrectly